src/compile_torture_tests.py

```diff
@@ -82,7 +89,7 @@ def run(c, cxx, testsuite, sysroot_dir, fails, out, config='wasm'):
   return result
 
 
-def getargs():
```

Comment:
Why not keep `getargs`? We can still wrap the call to `run` in a `main` function, but I think having one function do one thing is a useful property to keep.